_posts/2021-04-08-pulp-3.12-is-generally-available.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Pulp 3.12.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - announcement
+---
+
+Let's take a look at the main updates and enhancements as part of this release.
+
+This release also includes a plethora of bugfixes as well as some nice enhancements to Pulpcore and the Plugin API. For a full overview of changes, check out the [changlog](https://docs.pulpproject.org/pulpcore/changes.html).
+
+## Support added for automatic publishing and distributing [#7626](https://pulp.plan.io/issues/7626)
+
+When the content of a repository is updated, the repository can now be automatically published and redestributed.
```

Comment:
Should we mention, that this work needs to be picked up by upcoming plugin releases to be used?

Reply:
The fact that all the plugins have compatibility releases is a known fact, or you think I should be more specific? Doesn't this apply to all plugins and their ability to use enhancements in newer releases?

Reply:
To me the announcement here sounds like "You can use autopublish on _all_ your repos _now_."
But in fact, this lays the groundwork for plugins to support autopublish/-distribute.
So for expectation management, i would add something like:
"Look out for upcoming plugin releases to add this feature to their repositories."

Reply:
OK this makes sense. Hope my rephrasing is adequate.